spec/system/new_pattern_generator_spec.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe FenrirView::Generators::NewPatternGenerator, type: :system do
+  before(:context) do
+    system "cd test/dummy; rails g fenrir_view:new_pattern component widget"
```

Comment:
Yeah this is super gross! Googling "generator test" I fould https://github.com/stevehodgkiss/generator_spec ?

Reply:
Yeah, I think that's the most recently update one I found. So if I'm keeping the test in, I'd probably want to try that gem.

Neo suggested to maybe not test it at all, since it's only for internal use and all it does is copy some files at the moment.